src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Machine started its work.");

```

Comment:
The message should indicate that a `Truck` is working right now. Consider updating the message to be more informative and specific to the `Truck` class.